platform/ui/src/components/Table/Table.mdx

```diff
@@ -0,0 +1,124 @@
+---
+name: Table
+menu: Components
+route: components/table
+---
+
+import { Playground, Props, Link } from 'docz';
+import { Table, TableHead, TableBody, TableRow, TableCell } from '@ohif/ui';
+
+# Table
+
+Tables display sets of data.
+
+_The Table component and its components in the tree are built using divs._
+
+## Import
+
+Check the given example below to see how to import each component to create a
+table.
+
+```javascript
+import { Table, TableBody, TableCell, TableHead, TableRow } from '@ohif/ui';
+```
+
+## Basic usage
+
+<Playground>
+  <div className="p-4">
+    <Table>
+      <TableHead>
+        <TableRow>
```

Comment:
`[0, 1, 2, 3, 4, 5].map(x => ());` would condense the size of this when the example code is displayed on page. Lots of scrolling to get past:

![image](https://user-images.githubusercontent.com/5797588/77670975-4b0ce780-6f5d-11ea-97aa-e0ac86e69468.png)